cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/CloudControllerClient.java

```diff
@@ -1368,4 +1368,6 @@ UploadToken asyncUploadApplication(String applicationName, ApplicationArchive ar
     CloudBuild getBuild(UUID buildGuid);
 
     void bindDropletToApp(UUID dropletGuid, UUID applicationGuid);
+    
+    List<CloudBuild> getAllBuildsForApplication(UUID applicationGuid);
```

Comment:
Rename to `getBuildsForApplication` for consistency with `getDomainsForOrganization`.

Reply:
Done